views/forms/BarChartFields.jsx

```diff
@@ -118,6 +145,29 @@ const BarChartFields = ({
               <Delete className={classes.icon} />
             </Button>
           </div>
+          <div key={idx} className={classes.formLabelRow}>
```

Comment:
is there anything unique we can use for `key` that isn't an `index`?

Reply:
[d28f2d6](https://github.com/AMP-SCZ/dpdash/pull/110/commits/d28f2d6b1be0f5c994babe2ddbf5581cbeed790c)

Reply:
[This](https://github.com/AMP-SCZ/dpdash/pull/110/commits/d28f2d6b1be0f5c994babe2ddbf5581cbeed790c#diff-fbb0f165f85d2c475bce82a02e022f5ab33830668e956de1139b5eef41a55a13R113) is still using an index as the unique part of the key. My concern here is around performance, and if there is nothing on `field` we can use that is unique, then this is totally fine.

Here is the [React docs on keys](https://reactjs.org/docs/lists-and-keys.html#keys) and why we want to avoid using indices when possible

>We don’t recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. Check out Robin Pokorny’s article for an [in-depth explanation on the negative impacts of using an index as a key](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). If you choose not to assign an explicit key to list items then React will default to using indexes as keys.

Reply:
Right now there's nothing unique for field, so I've prepended `field-map` but still need the index to make it somewhat different.

Reply:
sounds good, you could omit the `field-map` portion if you want to